docs/advanced/atomic-swap.md

```diff
@@ -0,0 +1,38 @@
+# Atomic Swaps
+
+---
+
+## Overview
+
+An Atomic Swap is a smart contract technology which makes possible to exchange coins from two different blockchains without having to trust any third party, for example a centralized exchange. 
+For an atomic swap, a cryptocurrency must have the 'OP_CHECKLOCKTIMEVERIFY' opcode, which makes a timelock possible.
+The other prerequisite is that they need to have the same hash function. That doesn't necessarily mean that they use the same hash functions for mining, but they need to have the same hash function available in their transaction scripting language.
+
+---
+
+## History of Atomic Swaps
+
+Tier Nolan is considered the inventor of atomic swaps. The concept of decentralized exchanges were widely discussed already in 2012. [The first atomic swap](https://blog.decred.org/2017/09/20/On-Chain-Atomic-Swaps/) happened between Decred and Litecoin on September 20, 2017.
+
+## Using Decred's atomicswap tool
+
+To perform an atomic swap, one needs to use the necessary command line tools for both chains. Each blockchain has it’s own tools. 
+This are the steps for an atomic swap:
+ * One of the parties needs to *initiate* the process. This way a smart contract will be created. In this smart contract, a hash of a secret will be visible. The secret is only known by the initiator at this point. The secret is a very large number (32 byte). 
```

Comment:
Add a blank line before this to ensure the bullet point list is rendered correctly.